jyhton/src/org/python/ReL/ProcessLanguages.java

```diff
@@ -211,18 +211,21 @@ else if(rq.getAttributePath(j).levelsOfIndirection() == 0)
 
 	public synchronized void processNativeSIM(String ReLstmt) throws Exception {
 
-
 		boolean DBG = true;
-		OracleNoSQLDatabase db = (OracleNoSQLDatabase) connDatabase;
-		if (ReLstmt.equalsIgnoreCase("clear database")) {
-			db.clearDatabase();
-			return;
-		}
-		if (ReLstmt.equalsIgnoreCase("stop database")) {
-			db.ultimateCleanUp("Stop database");
-			return;
+		ReLstmt = ReLstmt.replaceAll("\\_\\^\\_", ";");
+
+		if (DBG) {
+			if (ReLstmt.equalsIgnoreCase("clear database")) {
```

Comment:
Whoops, this block of code should be indented. I'll correct it for the next pull as it doesn't affect the logic whatsoever.